### ISSUE TITLE
iOS platform fails to register plugin

### ISSUE THREAD

**Issue by EgilSandfeld** [resonance-audio/resonance-audio-wwise-sdk#3]
Hi again :)

This is a continuation of https://github.com/resonance-audio/resonance-audio-wwise-sdk/issues/1
I experience the exact same issue on a new project using

Wwise 2017.2.1
Unity 2017.3.0f3
Wwise Integration 2017.2.1
Resonance Audio 1.1.1

![](https://user-images.githubusercontent.com/5029330/35919020-096cc114-0c14-11e8-97e6-86c16fd4da86.jpg)

> Hi Egil,
> 
> I can see one potential step missing in your setup for the iOS build. Could you verify if you copy the ResonanceAudioFXFactory.h header file as well into the same folder that you copy the iOS library into (Assets/Wwise/Deployment/Plugins/iOS/DSP)?
> 
> That solves the registration issue for me.

I did exactly that for this project , Anokta. Issue persists though audio comes through all fine, it's only RA not working..

Let me know if you need further info! It might not be a bug on your end - it might as well be using Wwise 2017.2.x but I can't tell.

**Comment by EgilSandfeld** [comment]
I'll try RA 1.2.0 today and see if the issue persists.

**Comment by EgilSandfeld** [comment]
Trying 1.2.0 the first time didn't change the result after a fresh Xcode project. It still would give the "Could not register the plugin ..."

Then I discovered that I missed out setting a proper bundle id in the Unity Player Settings. I added one and rebuild a fresh Xcode. That actually worked!

I don't know how I could miss that, but I don't understand why that would make the difference really..
I'm happy it works, but as always is confused on the funny things that always happens building to iOS ;)

Thanks again guys for a great spatializer!
